### ISSUE TITLE
Session time and batch info

### ISSUE THREAD

**Issue by imouzon** [imouzon/dmc2015#1]
I added CSV with a new set of features to the features/feature_files folder.
The features for the training set are stored in ./dmc2015/features/feature_files/batchInfo_train.csv
The features for the test set are stored in ./dmc2015/features/feature_files/batchInfo_test.csv
Both files contain the corresponding orderID columns so that they can be joined to a larger feature matrix.
I have updated the feature description file with an explanation of what each variable means.
Thanks!
